editor/src/utils/canvas-react-utils.ts

```diff
@@ -143,11 +145,25 @@ function attachDataUidToRoot(
   }
 }
 
+const MeasureRenderTimes = !PRODUCTION_ENV && typeof window.performance.mark === 'function'
+
 const mangleFunctionType = Utils.memoize(
   (type: unknown): React.FunctionComponent => {
+    const uuid = v4()
```

Comment:
```suggestion
    const uuid = MeasureRenderTimes ? v4() : ''
```

Reply:
hmm, now you are saynig that, we should make a todo to measure the memoization of these mangled functions